feature_engine/features/politeness_features.py

```diff
@@ -0,0 +1,34 @@
+import convokit
+import spacy
+import pandas as pd
+import re
+
+from convokit import Corpus, Speaker, Utterance
+from convokit import download
+from convokit import TextParser
+
+from convokit import PolitenessStrategies
+
+'''
+function: politeness_features
+(Chat-level function)
+
+This gets the politeness annotations of each message, with some fields 
+including HASHEDGE, Factuality, Deference, Gratitude, Apologizing, etc.
+
+'''
+
+ps = PolitenessStrategies()
+'''
+Note: if you get an error in which `en_core_web_sm` is not found, do the following:
+python3 -m spacy download en_core_web_sm
+'''
+spacy_nlp = spacy.load("en_core_web_sm", disable=["ner"])
```

Comment:
This looks great! Awesome work, @agshruti12 !!! I ran everything on my end and just added a few more comments/documentation. the whole thing seems to run in about 6 minutes on my machine (~1 min per dataset), which is super reasonable. Hoorary for getting your first feature done (and it's really 20 feature in one)!